







int32 n_object



sensor_msgs/PointCloud[] cluster